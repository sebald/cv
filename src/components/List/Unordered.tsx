import { styled } from '../../style';

export const Unordered = styled.ul`
  margin: 0;
  padding: 0;

  list-style: none;
`;
