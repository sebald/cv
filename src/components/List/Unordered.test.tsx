import { shallow } from 'enzyme';
import * as React from 'react';

import { Unordered } from './Unordered';

test('render correct', () => {
  const wrapper = shallow(<Unordered />);
  expect(wrapper).toMatchSnapshot();
});
