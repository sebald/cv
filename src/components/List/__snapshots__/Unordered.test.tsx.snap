// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correct 1`] = `
.c0 {
  margin: 0;
  padding: 0;
  list-style: none;
}

<ul
  className="c0"
/>
`;
